### PR TITLE
lint changes

### PR DESCRIPTION
**Pull request by esharf**
## Status
- [x] Ready

## Description
changed the `devtestdemito` docker image creation.

before:
1. create an image using docker build from jinja file for python/pwsh dockers.
2. create an image using docker build to copy the pack files into the docker.

after:
1. create a container using the base image
2. run an installation script depending on the container type
3. commit the container into a new image
4. copy the files directly to the container without building

## Must have
- [ ] Tests

### CODE REVIEW

**Review by dorschw — CHANGES_REQUESTED** [review]

File: demisto_sdk/commands/lint/docker_helper.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        return mounts
+
+    @staticmethod
+    def copy_files(container: docker.models.containers.Container, files: List[Tuple[str, PATH_OR_STR]]):
+        """
+        Args:
+            container: the container object.
+            files: a list of (target path in container, source path in machine).
+        Returns:
+            a list of mounts
```
Comment (dorschw) [inline demisto_sdk/commands/lint/docker_helper.py:77]: please explain what this method does (its name suggests it copies files. from where to where? and if so, what `mounts` are returned?)

File: demisto_sdk/commands/lint/docker_helper.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        container: docker.models.containers.Container = init_global_docker_client().containers.create(
+            image=image, command=command, environment=environment, **kwargs)
+        if files_to_push and not mount_files:
+            Docker.copy_files(container, files_to_push)
+        return container
+
+    @staticmethod
+    def create_image(base_image: str, image: str, container_type: str = 'python', install_packages: Optional[List[str]] = None) -> docker.models.images.Image:
+        """
+        this function is used to create a new image of devtestsdemisto docker images.
```
Comment (dorschw) [inline demisto_sdk/commands/lint/docker_helper.py:105]: please explain the flow here.

**Review by dorschw — CHANGES_REQUESTED** [review]

File: demisto_sdk/commands/lint/docker_helper.py [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+            base_image(str): the base docker image e.g. demisto/python3:3.10.0.23456
+            image(str) the new image name to create e.g. devtestsdemisto/python3:3.10.0.23456-d41d8cd98f00b204e9800998ecf8427e
+            container_type(str): can be 'python' or 'powershell'
+            install_packages(list(str)): pip packages to install e.g ["pip='*'", "pytlint==1.2.3"]
+        Returns:
+            the new created image
+        Flow:
+            1. creating a container
+            2. runn the istallation scripts
+            3. commit the docker changes (e.g. new python package installed) to a new image
```
Comment (dorschw) [inline demisto_sdk/commands/lint/docker_helper.py:114]:
```suggestion
            1. creating a container using an existing image
            2. running the istallation scripts 
            3. committing the docker changes (installed packages) to a new local image
```

**Comment by dorschw** [comment]
Please edit the PR description, explaining what it is about for future devs.

**Review by yaakovi — APPROVED** [review]

**Pull request merged**